### ISSUE TITLE
sre_parse deprecated in Python 3.12.3

### ISSUE THREAD

**Issue by Konfekt** [vaeth/replacer#1]
This happens in https://github.com/vaeth/replacer/blob/ab03cfd5867c594f19cb94a841d4f010148f7da9/bin/replacer#L23
Does not seem to have an obvious workaround without creating dependencies...

**Comment by vaeth** [comment]
You can simply ignore the warning. In some future python versions it will not show up and the (much slower) workaround will be used automatically.

**Comment by vaeth** [comment]
You mentioned “without creating dependencies": do you actually know a dependency which can be used instead?

**Comment by Konfekt** [comment]
I do not think so, since this is a question of speed; maybe the regex module can help, but it seems that this is a use case where sre_parse outperforms other approaches; I am no Python regex expert.
